src/contexts/MessagesContext.tsx

```diff
@@ -0,0 +1,36 @@
+import {createContext, Dispatch, useReducer} from 'react';
+import {ChatMessage, SenderType} from "../types";
+import {MessageActions, messagesReducer} from "../reducers/MessagesReducer";
+
+const initialMessages: ChatMessage[] = [
+  {sender: SenderType.AI, content: "Your Chat With Interviewer Starts Here ..."}
+];
+export const MessagesContext = createContext<{
+  messages: ChatMessage[];
+  dispatch: Dispatch<MessageActions>;
+}>({
+  messages: initialMessages,
+  dispatch: () => null
+});
+
+MessagesContext.displayName = "MessagesContext";
+
+export function InterviewContextProvider({children}: React.PropsWithChildren<{}>) {
+  // Use the useReducer hook to manage the cart state
```

Comment:
what is cart state?

Reply:
Oops, I forgot to clean up some useless comments that were copied from the previous project. I will fix this later.